taxi/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver
+
+
+class DriverForm(forms.ModelForm):
+    MIN_LICENSE_CHARACTERS = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != DriverForm.MIN_LICENSE_CHARACTERS:
+            raise ValidationError(f"License has to consist only {DriverForm.MIN_LICENSE_CHARACTERS}")
+
+        if license_number[0:3].isupper() is False:
+            raise ValidationError("The first 3 characters have to be UpperCase")

```

Comment:
add here other validation under the condition of the task "First 3 characters are uppercase letters"